### ISSUE TITLE
Building the tool

### ISSUE THREAD

**Issue by soveran** [ostap/comp#38]
I'm using this makefile:

``` makefile
y.output:
    go tool yacc -o src/comp/y.go -p "comp_" src/comp/grammar.y

test: comp
    go test comp

comp: y.output
    go build ./src/comp
```

I think it may be useful to have it in the repo. Regardless, I think the `go build ./src/comp` step is missing in the README. I can submit a pull request either for adding that step to the README, or to provide a makefile with the corresponding modifications to the README.

**Comment by ostap** [comment]
It has been a while since I last looked at it. From what I understand, go tools evolved and the whole step of changing the GOPATH can be eliminated. This, however, involves changing the commands above:

```
$ echo $GOPATH
/home/ostap/gopath
$ pwd
/home/ostap/comp
$ go tool yacc -o src/comp/y.go -p "comp_" src/comp/grammar.y
$ go build -o comp src/comp/*.go
$ go test src/comp/*.go
```

I will gladly accept a pull request with the Makefile and a README update.

**Comment by soveran** [comment]
The tests don't pass for me, but I will send you the PR anyway so you can check what could be happening.

**Comment by ostap** [comment]
Indeed. It is related to #26. I temporarily disabled the tests which reproduce the issue.
